doc/source/performance/procfs-plugin.rst

```diff
@@ -0,0 +1,83 @@
+.. procfs-plugin:
+
+======================================================================
+Using the ProcFS plugin
+======================================================================
+
+
+Implemented in |Percona Server| 8.0.25-15, the ProcFS plugin provides access to the Linux performance counters by running SQL queries against a Percona Server for MySQL 8.0 (or other MySQL 8.0-compatible variant). This plugin is useful for Cloud and MySQL-as-a-service installations.
```

Comment:
use :rn:`8.0.25-15` so it automatically links to release notes. This helps user to see when (date) this has been released and what other features/bugs landed at the same time.

Reply:
Fixed.